xbmc/settings/dialogs/GUIDialogSettingsBase.cpp

```diff
@@ -539,6 +552,11 @@ std::set<std::string> CGUIDialogSettingsBase::CreateSettings()
   return settingMap;
 }
 
+string CGUIDialogSettingsBase::GetSettingsLabel(CSetting *pSetting)
```

Comment:
What exactly is the use case here? It's just an indirection and doesn't seem to be used anywhere else.